infrastructure/main.tf

```diff
@@ -49,3 +53,15 @@ module "service" {
     SERVICE_BUS_CONNECTION_STRING = "${module.servicebus-topic.primary_send_and_listen_connection_string}/subscriptions/${local.subscription_name}"
   }
 }
+
+module "key-vault" {
```

Comment:
we could name this something like `private-beta-app-vault` ?

Reply:
as long as there's only one vault used the current name is clear imho.
The names of other modules in this file are: `servicebus-namespace`,  `servicebus-topic` etc.

Reply:
It all lives in the context of private beta app.